cmd/deploy/remote.go

```diff
@@ -446,8 +446,11 @@ func (rd *remoteDeployCommand) getContextPath(cwd, manifestPath string) string {
 		return cwd
 
 	}
+	// if the manifest filepath is a directory, we need to set the parent directory as the build context
+	// Example: 'okteto deploy -f apps/service', the build context should be 'apps'
+	// because otherwise in the remote builder, we will not find the folder 'service' because we are inside of it
```

Comment:
@pchico83 How should behave this from a product perspective? If I execute the command from the project root using `okteto -f apps/service`, being `apps/service` the folder where the manifest is, which should be the context folder for the remote command?

I though we wanted to be the parent folder of the okteto manifest, but  I'm not sure

Reply:
From a product perspective, I would only consider `-f` with a path to the okteto manifest. 
But yes, the context should always be the folder where the okteto manifest is (except for `.okteto` folder). We should also support a `context` field in the deploy section, but that's a separate discussion